torch/_logging/_internal.py

```diff
@@ -644,7 +645,7 @@ def pad_to(s, length=30):
 
   TORCH_LOGS_OUT=/tmp/output.txt will output the logs to /tmp/output.txt as
   well. This is useful when the output is long.
-"""  # flake8: noqa: B950
```

Comment:
This should be probably just "noqa: B950".

"flake8: noqa: B950" actually disables everything flake8 for the whole file, and "noqa: B950" disables just rule B950 locally.

Reply:
Thank you for the advice! I can certainly do that

Reply:
I actually don't get an error for this when I remove the noqa which means it's unnecessary -- can I just remove the noqa entirely?

Reply:
Sure